domains/auth/Auth.API/API/Controllers/AuthController.cs

```diff
@@ -39,6 +48,35 @@ public AuthController(ILogger<AuthController> logger, IOidcProviders oidcProvide
         return _oidcProviders.CreateAuthorizationUri(state);
     }
 
+    [HttpPost]
+    [Route("/invalidate")]
+    public IActionResult Invalidate([FromQuery] string state)
+    {
+        AuthState? authState = null;
+        try
+        {
+            authState = JsonSerializer.Deserialize<AuthState>(
+                _cryptographyService.Decrypt(state),
+                new JsonSerializerOptions()
+                {
+                    DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
+                });
+        }
+        catch (Exception e)
+        {
+            return BadRequest("Cannot decrypt " + nameof(AuthState));
+        }
+
+        var validationResult = _validator.Validate(authState);
+        if (!validationResult.IsValid)
```

Comment:
```suggestion
        if (!_validator.Validate(authState).IsValid)
```

Reply:
Nice point. Though i kept it as we starting using `ValidationProblem` in 7c8c0e6 and therefore need to variable. :)